addons/advanced_fatigue/ACE_Settings.hpp

```diff
@@ -1,49 +0,0 @@
-class ACE_Settings {
```

Comment:
movedToSQF = 1 for settings moved to use CBA_settings_fnc_init?

Reply:
@jonpas do we still provide BWC in 3.13?